docs/conversion/s2u_features/S2U-5-oracle.sql

```diff
@@ -0,0 +1,3 @@
+-- S2U-5 --
+ALTER TABLE rbc_rubric ADD adhoc NUMBER(1) DEFAULT 0;
```

Comment:
```suggestion
ALTER TABLE rbc_rubric ADD adhoc NUMBER(1, 0) DEFAULT 0 NOT NULL;
```

Reply:
But it can be null, all the existing entries will be null...